test/e2e/bootstrapper_test.go

```diff
@@ -180,6 +185,23 @@ func getSvcInfo(svcName string) (svc.State, string, []string, error) {
 	}
 }
 
+// getSvcConfig returns the configuration of the Windows Service with the given name
+func getSvcConfig(svcName string) (mgr.Config, error) {
+	svcMgr, err := mgr.Connect()
+	if err != nil {
+		return mgr.Config{}, fmt.Errorf("could not connect to Windows SCM: %s", err)
+	}
+	defer svcMgr.Disconnect()
+
+	svcHandle, err := svcMgr.OpenService(svcName)
+	if err != nil {
+		return mgr.Config{}, err
+	}
+	defer svcHandle.Close()
```

Comment:
This block can be modularized. See lines [160-170](https://github.com/openshift/windows-machine-config-bootstrapper/pull/315/files#diff-2f248ac78cf47e8f98e4c678f4208351bc12c9da943275b4be0a3ce500e31bfeR160-R170)

Reply:
That change would require a larger refactor, across all the `getSvc*` functions, I thought about doing it, but decided against it, especially considering that this code will be taken out of service soon.